internal/broadcast/definition.go

```diff
@@ -25,28 +25,31 @@ import (
 	"github.com/hyperledger/firefly/pkg/fftypes"
 )
 
-func (bm *broadcastManager) BroadcastDefinitionAsNode(ctx context.Context, ns string, def fftypes.Definition, tag fftypes.SystemTag, waitConfirm bool) (msg *fftypes.Message, err error) {
-	return bm.BroadcastDefinition(ctx, ns, def, &fftypes.Identity{ /* resolve to node default */ }, tag, waitConfirm)
+func (bm *broadcastManager) BroadcastDefinitionAsNode(ctx context.Context, ns string, def fftypes.Definition, tag string, waitConfirm bool) (msg *fftypes.Message, err error) {
+	return bm.BroadcastDefinition(ctx, ns, def, &fftypes.SignerRef{ /* resolve to node default */ }, tag, waitConfirm)
 }
 
-func (bm *broadcastManager) BroadcastDefinition(ctx context.Context, ns string, def fftypes.Definition, signingIdentity *fftypes.Identity, tag fftypes.SystemTag, waitConfirm bool) (msg *fftypes.Message, err error) {
+func (bm *broadcastManager) BroadcastDefinition(ctx context.Context, ns string, def fftypes.Definition, signingIdentity *fftypes.SignerRef, tag string, waitConfirm bool) (msg *fftypes.Message, err error) {
 
-	err = bm.identity.ResolveInputIdentity(ctx, signingIdentity)
+	err = bm.identity.ResolveInputSigningIdentity(ctx, ns, signingIdentity)
 	if err != nil {
 		return nil, err
 	}
 
 	return bm.broadcastDefinitionCommon(ctx, ns, def, signingIdentity, tag, waitConfirm)
 }
 
-func (bm *broadcastManager) BroadcastRootOrgDefinition(ctx context.Context, def *fftypes.Organization, signingIdentity *fftypes.Identity, tag fftypes.SystemTag, waitConfirm bool) (msg *fftypes.Message, err error) {
+func (bm *broadcastManager) BroadcastDefinitionResolveKeyOnly(ctx context.Context, ns string, def fftypes.Definition, signingIdentity *fftypes.SignerRef, tag string, waitConfirm bool) (msg *fftypes.Message, err error) {
 
-	signingIdentity.Author = bm.identity.OrgDID(def)
+	signingIdentity.Key, err = bm.identity.ResolveInputSigningKeyOnly(ctx, signingIdentity.Key, true)
```

Comment:
Should we clear `signingIdentity.Author` here? Or perhaps we should just pass in a key, instead of passing in a `SignerRef` that is implied to have a key with no author.

Reply:
The intention of this function, is that it doesn't mess with the `Author` - because that must in the case of an `IdentityClaim` match the identity being created. So it's the caller's responsibility to have that set before calling in here. However, this function still gets the identity plugin to "resolve" the key (see comment above about spelling) - such as getting Fabric to look up the full MSP Key Identifier for a shortname.

Think I'll just add comments if that's ok, but please feel free to come back around if that doesn't help.

Reply:
Resolved with the `normalization` rename